src/components/PeoplePage/index.tsx

```diff
@@ -0,0 +1,205 @@
+import { useEffect, useState } from 'react';
+import { Loader } from '../Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(true);
+
+  const { personInfo } = useParams();
+
+  useEffect(() => {
+    getPeople()
+      .then(setPeople)
+      .catch(() => setErrorMessage('Something went wrong'))
+      .finally(() => setLoading(false));
+  });
```

Comment:
```suggestion
  useEffect(() => {
    getPeople()
      .then(setPeople)
      .catch(() => setErrorMessage('Something went wrong'))
      .finally(() => setLoading(false));
  }, []);
```

Reply:
Be careful with that 🙃

https://github.com/user-attachments/assets/d35dd578-8eb2-4be5-874c-078b64214088